tests/mssql.yaml

```diff
@@ -19,7 +19,8 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mssql
-        image: mcr.microsoft.com/mssql/server:2017-latest
+        #image: mcr.microsoft.com/mssql/server:2017-latest
+        image: quay.io/mkarg/mssql:latest
```

Comment:
Can we look at moving this image into cloud-bulldozer org in quay, also can you put the dockerfile for the image into snafu or somewhere in the cb-ecosystem

Reply:
We can put the dockerfile in snafu, but this image includes a database that is pre-initialized, which I don't think we can (easily) do directly in the dockerfile, so the image may need to be manually maintained instead of through the github>quay automation.

Reply:
I tried that and found no working solution as we'd need the DB to be running in order to create the schema from the Dockerfile. If somebody has an idea I'm happy to try it, but from what I can tell it's not possible.